src/solver/conv_asm_implicit_gemm_gtc_wrw_nhwc.cpp

```diff
@@ -644,6 +644,15 @@ void PerformanceConfigAsmImplicitGemmGTCWrwXdlopsNHWC::HeuristicInit(
     }
     else
     {
+        /// \todo Fix this
+        /// Clang-Tidy assumption:
+        /// (n_per_block == 0)  ==>
+        /// (n_per_block == config.gemm_n_per_block) ==>
+        /// (config.gemm_n_per_block == 0)  ==>
+        /// division by zero in GetImplicitGemmGtcDynamicWrwXdlopsNHWCKernel()
+        if(n_per_block == 0)
+            MIOPEN_THROW(miopenStatusInternalError);
+
```

Comment:
[R]
```suggestion
        if(n_per_block == 0) // For clang-tidy.
            MIOPEN_THROW(miopenStatusInternalError);

```

Reply:
This one is useful and can be fixed, so a little explanation would be nice.

Reply:
[R] Let's try to stay brief. See updated suggestion above.